src/object-observer.js

```diff
@@ -93,7 +97,15 @@ const
 		let l = observers.length;
 		while (l--) {
 			try {
-				observers[l](changes);
+				const target = observers[l]
+				const { path } = target.options
+				
+				if (path && path.split(".").every((value, index) => changes[0].path[index] === value)) {
```

Comment:
each change should be tested and decided upon - to be sent or not, right?
another thing - seems to me that joining the changes[i].path and verifying that it equals to `path` or starts from `path + .` would be faster, WDYT?

Reply:
Yes, i agree with you.
Made some changes, new filtering method looks like this,

```JS
if (path) {
					const containsPath = changes.filter((change) => change.path.join(".").startsWith(path))				
					if( containsPath.length > 0 ) {
						target.observer(changes);
					} 						
				} 
```
If it looks okay, i will make a new pull request for it.